misc/ripunzip/install.sh

```diff
@@ -2,7 +2,16 @@
 
 set -eu
 
-. misc/bazel/runfiles.sh
+# --- begin runfiles.bash initialization v3 ---
+# Copy-pasted from the Bazel Bash runfiles library v3.
+set -uo pipefail; set +e; f=bazel_tools/tools/bash/runfiles/runfiles.bash
+source "${RUNFILES_DIR:-/dev/null}/$f" 2>/dev/null || \
+  source "$(grep -sm1 "^$f " "${RUNFILES_MANIFEST_FILE:-/dev/null}" | cut -f2- -d' ')" 2>/dev/null || \
+  source "$0.runfiles/$f" 2>/dev/null || \
+  source "$(grep -sm1 "^$f " "$0.runfiles_manifest" | cut -f2- -d' ')" 2>/dev/null || \
+  source "$(grep -sm1 "^$f " "$0.exe.runfiles_manifest" | cut -f2- -d' ')" 2>/dev/null || \
+  { echo>&2 "ERROR: cannot find $f"; exit 1; }; f=; set -e
+# --- end runfiles.bash initialization v3 ---
```

Comment:
bleurgh, I wanted to avoid this... couldn't we just list our two possibilities?

```bash
source misc/bazel/runfiles.sh || soure ql~/misc/bazel/runfiles.sh  # or whatever
```

Reply:
ah, there's the buildfier thing. I would just remove the `dev_dependency=True` to that. In any case `codeql` is not meant to be used as a dependency for anything else than `semmle-code`. Also, the correct suggestion here is
```bash
source misc/bazel/runfiles.sh 2>/dev/null || source external/ql~/misc/bazel/runfiles.sh
```

Reply:
but I'd be ok if you insist :slightly_smiling_face:

Reply:
As you delivered the correct path, I'll fix the buildifier thing :D 
It is another dependency on tilde versus plus we'll have to fix, but oh well.